corpus/templates/base.html

```diff
@@ -4,19 +4,109 @@
 <html lang="en">
 
 <head>
+
+    <!-- Meta tags-->
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <!-- Give a small description in 3-4 sentences about the website -->
+    <meta name="description"
+        content="IEEE NITK is a local chapter of the IEEE Organisation belonging in the Bangalore R10 Section. IEEE NITK undertakes lots of projects by its various SIGs which are formed and projects undertaken by students are futher applied to help solve daily world problems.">
+    <!-- A small abstract idea about the website -->
+    <meta name="abstract" content="This Website is about IEEE NITK Student Chapter">
+    <!-- Important Keywords to be noted in the website  -->
+    <meta name="keywords" content="IEEE, IEEE NITK, NITK Surathkal, NITK, Clubs in NITK, Technical Clubs in NITK">
+    <!-- Tell the spider to index the first page and other pages as well-->
+    <meta name="robots" content="index, follow">
+    <!-- How often should spiders come back to your page -->
+    <meta name="revisit-after" content="3 days">
+
+    <!-- Copyright regarding the website -->
+    <meta name="copyright" content="IEEE NITK">
+    <!-- Tells Google Bot not to duplicate description -->
+    <meta name="googlebot" content="noodp">
+    <!-- Language for the website -->
+    <meta name="language" content="English">
+
+    <!-- Web Author for the image -->
+    <meta name="web_author" content="IEEE NITK">
+    <meta name="author" content="IEEE NITK">
+    <!-- Email ID -->
+    <meta name="contact" content="corpusieeenitk@gmail.com" />
+    <!-- Email ID to reply to -->
+    <meta name="reply-to" content="corpusieeenitk@gmail.com">
+
+    <!-- Refers to distribution of the page -->
+    <meta name="distribution" content="global">
+    <!-- Generator or formatter tag -->
+    <meta name="generator" content="Jekyll">
+    <!-- Disallow spammers for the webpage -->
+    <meta name="no-email-collection" content="http://www.metatags.info/nospamharvesting">
+    <!-- Rating for the page -->
+    <meta name="rating" content="general">
+    <!-- Content Type for the page -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <!-- Fixing viewport on mobile views -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1.0, user-scalable=no">
+
+    <!-- Google Authorship Markup -->
+    <!-- Social: Twitter -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@{{site.twitter_username}}">
+    <meta name="twitter:title" content="{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}">
+    <meta name="twitter:description"
+        content="{% if post.description %}{{ post.description|striptags|linebreaksbr|truncatechars:160 }}{% else %}{{ site_description }}{% endif %}">
+    <meta property="twitter:image:src"
+        content="{% static 'img/blog-image.png' %}">
```

Comment:
We don't have access to IEEE NITK's Twitter anymore. Delete these tags.

Reply:
Done, also optimized Google+ tags along with Facebook / Open Graph tags...